frontend/src/dialog/photo/edit/details.vue

```diff
@@ -469,6 +471,27 @@ export default {
 
       this.time = taken.toFormat("HH:mm:ss");
     },
+    pastePosition(event) {
+      /* If text is pasted in the form of "XXX.XXXX, YYY.YYYY" (for example if
+       * pasting a position from external mapping app) then auto-fill both lat
+       * and lng fields
+       */
+      let paste = (event.clipboardData || window.clipboardData).getData("text");
+      let s = paste.split(/[ ,]+/);
```

Comment:
Looks like it might throw an exception if paste isn't a string for some reason?

Reply:
The specification of this seems to indicate that you will always get an empty string result even if the data type does not support strings (https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/getData#return_value).

I tested this by pasting some non-text data (eg image data), and the code just ends up with s.length == 0, and thus ignoring it.